Sources/Operator/abstract_operator.cc

```diff
@@ -44,7 +57,7 @@ void AbstractOperator::ForEachConn(VectorConstRefType v,
 }
 
 Eigen::VectorXcd LocalValues(Eigen::Ref<const RowMatrix<double>> samples,
-                             Eigen::Ref<const Eigen::VectorXcd> values,
+                             Eigen::Ref<const Eigen::VectorXcd> /*log_values*/,
```

Comment:
If we do not use the `log_values`, we should remove this parameter, otherwise it is confusing.